lib/puppet/provider/user/hpux.rb

```diff
@@ -19,14 +22,23 @@
     value !~ /\s/
   end
 
-  has_features :manages_homedir, :allows_duplicates, :manages_passwords
+  has_features :manages_homedir, :allows_duplicates, :manages_passwords, :manages_password_age, :manages_expiry
 
   def deletecmd
     super.insert(1,"-F")
   end
 
   def modifycmd(param,value)
-    super.insert(1,"-F")
+     cmd = super(param, value)
+     cmd << "-F"
+     if self.trusted == "Trusted"
+        cmd << ";"
```

Comment:
What is this doing? Why use a semi-colon - which appears to just ignore the 1st command - instead of just returning the modprpw call?